terraform-plans/configs/charm-prometheus-libvirt-exporter_main.tfvars

```diff
@@ -26,7 +26,10 @@ templates = {
       # We prefer the github runners because they are smaller machines and save resources.
       # If we have issues with it, we can switch to the larger and more numerous self-hosted options:
       # - runs-on: [self-hosted, jammy, ARM64]
-      runs_on            = "[[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]",
+      #
+      # Cannot test on s390x because setup-python action does not support s390x
```

Comment:
@chanchiwai-ray do we have a follow up ticket or issue for replacing setup-python in our workflows?

Reply:
https://github.com/canonical/solutions-engineering-automation/issues/206